documentation/proc-pages/physics-models/fusion_reactions/plasma_reactions.md

```diff
@@ -0,0 +1,236 @@
+- `self.proton_rate_density`: Proton production rate density.
+
+-----------------------
+
+### Sum the fusion rates | `sum_fusion_rates()`
+
+This method updates the cumulative class fusion power densities and reaction rates for alpha particles, charged particles, neutrons, and protons.
+
+#### Parameters:
+- `alpha_power_add` (float): Alpha particle fusion power per unit volume [MW/m³].
+- `charged_power_add` (float): Other charged particle fusion power per unit volume [MW/m³].
+- `neutron_power_add` (float): Neutron fusion power per unit volume [MW/m³].
+- `fusion_rate_add` (float): Fusion reaction rate per unit volume [reactions/m³/s].
+- `alpha_rate_add` (float): Alpha particle production rate per unit volume [/m³/s].
+- `proton_rate_add` (float): Proton production rate per unit volume [/m³/s].
+
+The above input values are added to the current object values for their respective power density.
+
+-----------------------
+
+### Calculate fusion rates | `calculate_fusion_rates()`
+
+This runner function is called to run all 4 fusion reaction calculation functions.
+
+This method sequentially calculates the fusion reaction rates and power densities for the following reactions:
+- Deuterium-Tritium (D-T)
+- Deuterium-Helium-3 (D-3He)
+- Deuterium-Deuterium (D-D) first branch
+- Deuterium-Deuterium (D-D) second branch
+
+It updates the instance attributes for the cumulative power densities and reaction rates for alpha particles, charged particles, neutrons, and protons.
+
+-----------------------
+
+### Set global physics variables | `set_physics_variables()`
+
+This method sets the required physics variables in the `physics_variables` and `physics_module` modules. It updates the global physics variables and module variables with the current instance's fusion power densities and reaction rates.
+
+#### Updates:
+- `physics_variables.alpha_power_density_plasma`: Updated with `self.alpha_power_density`
+- `physics_variables.charged_power_density`: Updated with `self.charged_power_density`
+- `physics_variables.neutron_power_density_plasma`: Updated with `self.neutron_power_density`
+- `physics_variables.fusion_rate_density_plasma`: Updated with `self.fusion_rate_density`
+- `physics_variables.alpha_rate_density_plasma`: Updated with `self.alpha_rate_density`
+- `physics_variables.proton_rate_density`: Updated with `self.proton_rate_density`
+- `physics_module.sigmav_dt_average`: Updated with `self.sigmav_dt_average`
+- `physics_module.dt_power_density_plasma`: Updated with `self.dt_power_density`
+- `physics_module.dhe3_power_density`: Updated with `self.dhe3_power_density`
+- `physics_module.dd_power_density`: Updated with `self.dd_power_density`
+- `physics_variables.f_dd_proton_branching`: Updated with `self.f_dd_proton_branching`
+
+-----------------------
+
+## Coupled alpha particle power
+
+!!! quote "ITER Physics Expert Group on Energetic Particles, Heating and Current Drive"
```

Comment:
Not sure quote is actually needed when you reference the source anyway in your statement below when you say assumed ITER default

Reply:
Thought it was nice to add a bit of source context